src/ucp/api/ucp.h

```diff
@@ -138,19 +138,33 @@ enum ucp_params_field {
  * during @ref ucp_init "UCP initialization" process.
  */
 enum ucp_feature {
-    UCP_FEATURE_TAG          = UCS_BIT(0),  /**< Request tag matching
-                                                 support */
-    UCP_FEATURE_RMA          = UCS_BIT(1),  /**< Request remote memory
-                                                 access support */
-    UCP_FEATURE_AMO32        = UCS_BIT(2),  /**< Request 32-bit atomic
-                                                 operations support */
-    UCP_FEATURE_AMO64        = UCS_BIT(3),  /**< Request 64-bit atomic
-                                                 operations support */
-    UCP_FEATURE_WAKEUP       = UCS_BIT(4),  /**< Request interrupt
-                                                 notification support */
-    UCP_FEATURE_STREAM       = UCS_BIT(5),  /**< Request stream support */
-    UCP_FEATURE_AM           = UCS_BIT(6)   /**< Request Active Message
-                                                 support */
+    /**< Request tag matching support */
+    UCP_FEATURE_TAG           = UCS_BIT(0),
+
+    /** Request remote memory access support */
+    UCP_FEATURE_RMA           = UCS_BIT(1),
+
+    /** Request 32-bit atomic operations support */
+    UCP_FEATURE_AMO32         = UCS_BIT(2),  
+
+    /** Request 64-bit atomic operations support */
+    UCP_FEATURE_AMO64         = UCS_BIT(3),  
+
+    /** Request interrupt notification support */
+    UCP_FEATURE_WAKEUP        = UCS_BIT(4),  
+
+    /** Request stream support */
+    UCP_FEATURE_STREAM        = UCS_BIT(5),
+
+    /** Request Active Message support */
+    UCP_FEATURE_AM            = UCS_BIT(6),  
+
+    /**
+     * Request selection of transports with capability to perform local
+     * operations using a peer's memory buffer and memory handle associated
+     * with it, and created with @ref UCP_MEM_MAP_EXPORTED capability.
```

Comment:
i'd rephrase like:
`operations using a peer's memory buffer and memory handle created with @ref UCP_MEM_MAP_EXPORTED capability`

Reply:
done